base/applications/kbswitch/kbswitch.c

```diff
@@ -28,14 +28,64 @@
 
 PKBSWITCHSETHOOKS    KbSwitchSetHooks    = NULL;
 PKBSWITCHDELETEHOOKS KbSwitchDeleteHooks = NULL;
-UINT ShellHookMessage = 0;
+UINT      ShellHookMessage      = 0;
+HINSTANCE hInst                 = NULL;
+HANDLE    hProcessHeap          = NULL;
+HMODULE   g_hHookDLL            = NULL;
+ULONG     ulCurrentLayoutNum    = 1;
+HICON     g_hTrayIcon           = NULL;
+HWND      g_hwndLastActive      = NULL;
+DWORD     gdwCodePageBitField   = 0;
+HWND      ghTrayWnd             = NULL;
+
+static VOID GetTrayWnd(VOID)
```

Comment:
Not a great function naming (a "Get" function is expected to return something, instead of void).
What about doing `ghTrayWnd = GetTrayWnd();` in both cases (below)?

Reply:
Fixed.